app/javascript/components/widgets/widgets/forest-change/fao-deforest/selectors.js

```diff
@@ -70,10 +71,13 @@ export const getSentence = createSelector(
       location: currentLabel,
       year: period && period.label,
       rate:
-        currentLabel === 'global'
-          ? `${format('.3s')(totalDeforest)}ha/yr`
-          : `${format('.3s')(deforest)}ha/yr`,
-      human: `${format('.3s')(humdef)}ha/yr`
+        rate < 1
+          ? `${format('.3r')(rate)}ha/yr`
```

Comment:
A simpler way would be to create a const which find the '.3r/s' and then insert that into the format. But it isnt important.